src/protocol/graphics.ts

```diff
@@ -122,6 +125,100 @@ function onMouseEvents(events: MouseEvent[], store: UIStore) {
   store.dispatch(actions.setEventsForType(gMouseClickEvents, "mousedown"));
 }
 
+export const Video = {
+  video: null as HTMLVideoElement | null,
+  // buffer: [] as string[],
+  all: new Uint8Array(),
+  blob: null as Blob | null,
+  videoReadyCallback: null as ((video: HTMLVideoElement) => any) | null,
+  commands: null as Promise<HTMLVideoElement> | null,
+  // mediaSource: new MediaSource(),
+  // sourceBuffer: null as SourceBuffer | null,
+  init() {
+    this.video = document.querySelector("#graphicsVideo");
+    this.setSrc();
+    // if (this.video && !this.video.src) {
+    //   this.video.src = URL.createObjectURL(this.mediaSource);
+    //   this.mediaSource.addEventListener("sourceopen", () => {
+    //     this.sourceBuffer = this.mediaSource.addSourceBuffer('video/webm; codecs="vp9"');
+    //     // this.sourceBuffer.mode = "sequence";
+    //     this.sourceBuffer.addEventListener("onerror", console.error);
+    //     // Flush the next fragment on the buffer as soon as the last update finishes
+    //     this.sourceBuffer.addEventListener("updateend", () => this.mediaSource.endOfStream());
+    //     // Try to flush the first buffered fragment if it exists
+    //     this.flush();
+    //   });
+    // }
+  },
+  onVideoEvent(fragment: string, store: UIStore) {
+    this.render(fragment);
+  },
+  render(fragment: string) {
```

Comment:
we could remove `onVideoEvent()` and `render()` and just use `append()`

Reply:
I left these in intentionally to aid in migrating to the original architecture using `MediaSource` but I'm content to remove them.